### ISSUE TITLE
[v3.2.0-beta.44] Type vom DP wird falsch zurückgegeben

### ISSUE THREAD

**Issue by mcuiobroker** [Zefau/ioBroker.jarvis#2701]
Geräte DP
![image](https://github.com/Zefau/ioBroker.jarvis/assets/84818934/a48345fb-e0be-4022-b07f-0ff68a71661a)

Zugehöriger DP

```
{
  "common": {
    "read": true,
    "write": true,
    "name": "Jahr auswählen",
    "type": "string",
    "role": "state",
    "def": "2024",
    "states": {
      "2019": "2019",
      "2020": "2020",
      "2021": "2021",
      "2022": "2022",
      "2023": "2023",
      "2024": "2024",
      "2025": "2025",
      "2026": "2026",
      "": "auswählen"
    }
  },
  "native": {},
  "type": "state",
  "from": "system.adapter.javascript.0",
  "user": "system.user.admin",
  "ts": 1719941140109,
  "_id": "0_userdata.0.schoolfree.year",
  "acl": {
    "object": 1636,
    "state": 1636,
    "owner": "system.user.admin",
    "ownerGroup": "system.group.administrator"
  }
}
```

ioBroker-LOG

![image](https://github.com/Zefau/ioBroker.jarvis/assets/84818934/6520436b-08b8-46c6-990c-532fb67103f2)

**Comment by Zefau** [comment]
`number` bei einer Jahreszahl erscheint mir korrekt? jarvis kennt den Typ des Objekts nicht, sondern schickt den Typen wie er intern erkannt wird.

**Comment by mcuiobroker** [comment]
Nein, er sollte immer nur den Type zu geben, den er aus ioBroker bekommen hat, sonst bekommt man als User dauernd die Fehlermeldungen, da iobroker es prüft.
In diesem Fall (eigener DP) kann ich natürlich den Type anpassen, aber grundsätzlich wäre es besser, wenn jarvis das Objekt einliest, denn die Diskussion hatten wir schon öfter?

**Comment by Zefau** [comment]
Wenn ich alle Objekte zusätzlich zu den States einlese wird jarvis viel zu inperformant.

**Comment by mcuiobroker** [comment]
Kann man das nicht im Hintergrund machen, denn es wird ja nicht innerhalb der ersten 5 Sekunden ein Wert gesetzt.
Ausserdem sind die Daten doch sowieso im Cache, d.h. es wäre dann nur beim ersten Start (Cache ist leer) einzulesen.

**Comment by mcuiobroker** [comment]
Also die subScriptions werden ja pro TAB aufgebaut -> window.Socket.stateSubscriptions , wenn ich dann die dazugehörigen Objekte hole sind das nicht mal 500ms. Das merkt man bei Umschalten zum TAB nicht mal.
